src/Compilers/Test/Utilities/CSharp/CSharpTestBase.cs

```diff
@@ -515,6 +516,16 @@ public static SyntaxTree ParseWithRoundTripCheck(string text, CSharpParseOptions
                 options = options.WithConcurrentBuild(false);
             }
 
+#if TEST_OPERATION
```

Comment:
Consider renaming the flag--"Test Operation" sounds very general. "Test_IOperation"? "Test_IOperation_Support"?